cdk-v2/.cdk

```diff
@@ -1,2 +1,3 @@
 openshift.auth.scheme=Basic
 openshift.auth.username=test-admin
+cdk.version=2.0.0
```

Comment:
So one might as well hard code the version in the server adapter. In the long run we need https://github.com/projectatomic/vagrant-adbinfo/issues/64